virttest/qemu_devices/qcontainer.py

```diff
@@ -3105,16 +3105,20 @@ def _emulator_setup(binary, extra_options=None):
             if tpm_version in ('2.0',):
                 setup_cmd += ' --tpm2'
 
-            tpm_path = os.path.join(tmp_dir, '%s_%s_tpm_state_%s' %
-                                    (self.vmname, name, random_str))
-            if os.path.exists(tpm_path):
-                shutil.rmtree(tpm_path, ignore_errors=True)
-            os.makedirs(tpm_path)
-            setup_cmd += " --tpm-state %s" % tpm_path
+            tpm_state_path = params.get(
+                "tpm_state_path", os.path.join(
+                    tmp_dir, '%s_%s_tpm_state_%s' % (
+                        self.vmname, name, random_str)))
+            tpm_overwrite = params.get_boolean("tpm_overwrite")
+            if not os.path.exists(tpm_state_path):
+                os.makedirs(tpm_state_path)
```

Comment:
From the document of man:
```
       --overwrite
           Overwrite existing TPM state. All previous state will be erased.
           If this option is not given and an existing state file is found,
           an error code is returned.
```
For the condition of `tpm_overwrite is None`, I afraid it will raise errors if there is an existing file is found.
CC @qcheng-redhat @zhenyzha

Reply:
`tpm_overwrite` should not be `None`, `params.get_boolean("tpm_overwrite")` just returns bool values.

Reply:
> `tpm_overwrite` should not be `None`, `params.get_boolean("tpm_overwrite")` just returns bool values.


Yes, you are right, I made a mistake. It should:
`For the condition of "tpm_overwrite is True", I afraid it will raise errors if there is an existing file is found.`

Reply:
Oh yes, because I forgot to append this option, only handled the `--not-overwrite`, will add it.